x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/all/exceptions/exceptions_table.tsx

```diff
@@ -391,6 +393,10 @@ export const ExceptionListsTable = React.memo(() => {
               noItemsMessage={emptyPrompt}
               onChange={handlePaginationChange}
               pagination={paginationMemo}
+              onClick={() => {
+                // TODO: need the clickedItem
```

Comment:
Is this TODO related to the work @dhurley14 is doing?

Reply:
Exactly !, I added this line so we can test the UI until the Exceptions List get ready